widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java

```diff
@@ -30,16 +31,26 @@ public void setUseOverlay(boolean enabled) {
         this.useOverlay = enabled;
     }
 
+    public void setLegacyCompanyName(String companyName) {
+        this.legacyCompanyName = companyName;
+    }
+
     /** @noinspection StatementWithEmptyBody*/
     public String getCompanyName() {
         String remoteCompanyName = fetchRemoteCompanyName();
         boolean isCompanyNameSetFromGliaHub = remoteCompanyName != null && !remoteCompanyName.trim().isEmpty();
 
+        Logger.d("CONFIGURATION", "companyName =" + companyName);
+        Logger.d("CONFIGURATION", "legacyCompanyName =" + legacyCompanyName);
```

Comment:
Can these logs be removed or you think we need them?

Reply:
Removed testing logs